components/drops/useDrops.ts

```diff
@@ -115,11 +114,12 @@ export const useDropStatus = (id: string) => {
   }
 }
 
+const MINIMUM_FUNDS_ROUND = 4
```

Comment:
This should be unified across the app imo cc @exezbcz

Reply:
breaking?

Reply:
I think not